### PR TITLE
feat(projects): #1580 added updatedOn time for pull request

### LINKED ISSUE #1580: Pull request list in project dashboard to show last updated as well as when created
Currently showing when PR was created, it will be useful to show when last updated, to match the PR events 

Screenshot 2019-10-26 17 36 58

### PR DESCRIPTION
**Pull request by webkhushboo**
closes #1580 

### Notes

A summary of what was achieved in this PR

- [ ] UI: Show the updated on time for Pull Request
- [ ] UI: Show the created on time for Issue

image

### CODE REVIEW

File: web/src/app/projects/repository/repository.component.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 </div>
 <mat-accordion class="repository" *ngIf="repository">
   <mat-expansion-panel hideToggle #expansion [expanded]="true">
-    <mat-expansion-panel-header [collapsedHeight]="headerHeight+'px'" [expandedHeight]="headerHeight+'px'"
+    <mat-expansion-panel-header [collapsedHeight]="headerHeight+'px'" [expandedHeight]="headerHeight+'px'"
```
Comment (eddiejaoude) [inline web/src/app/projects/repository/repository.component.html]: Extra space at end of the line is not needed? There are few of these below

**Comment by eddiejaoude** [comment]
Is it possible to do the same for Issues?

File: web/src/app/projects/repository/repository.component.html (hunk 1 above)
Comment (webkhushboo) [reply web/src/app/projects/repository/repository.component.html]: Actually when I did `Alt+Shift+F` , it has automatically formatted the files and moved the long lines in new lines . When I checked the difference in VS Code , I fixed the difference and I couldn't see the extra spacing diff in my VS code. So it occurred.
I have fixed those diff by comparing in github . I can't see them in my visual code yet

Comment (eddiejaoude) [reply web/src/app/projects/repository/repository.component.html]: Oh ok 🤔

**Comment by webkhushboo** [comment]
@eddiejaoude 
Please remove this commit while merge this PR 
https://github.com/DashboardHub/PipelineDashboard/pull/1585/commits/58ca84bf62156eb332ec2906e4dc57ad43a2f6fa

**Comment by webkhushboo** [comment]
@eddiejaoude 

Done changes for issues too

Desktop:
image

Mobile:
image

File: web/src/app/projects/repository/repository.component.html [hunk 2]
```diff
@@ -85,10 +85,20 @@
                 <img matListAvatar [src]="issue.owner.avatarUrl" [alt]="issue.owner.username" />
                 <h4 mat-line class="repository-detail-card__content__title repository-detail-card__content__title__issues" fxLayoutGap="5px">
                   <a href="{{ issue.url }}">#{{ issue.number }}</a> {{ issue.title }}</h4>
+                <p class="repository-detail-card__content__time-container">                
+                  <span class="repository-detail-card__content__time-container-top">
+                 <div>
```
Comment (eddiejaoude) [inline web/src/app/projects/repository/repository.component.html]: Indentation looks a bit odd here

File: web/src/app/projects/repository/repository.component.scss [hunk 3]
```diff
@@ -123,29 +128,32 @@
...
-          position: absolute;
           font-size: 9pt;
           right: 0;
           top: 0;
         }
 
         &__time-container-bottom {
           width: 120px;
-          display: flex;
+          display: block;
```
Comment (eddiejaoude) [inline web/src/app/projects/repository/repository.component.scss:156]: I think this is causing the layout to break, I will put screenshots in the PR comments

**Comment by eddiejaoude** [comment]
When resizing to smallest screen size the layout seems to be a bit weird...

## Large

Screenshot 2020-01-03 08 00 55

## Medium

Screenshot 2020-01-03 08 00 28

## Small

Screenshot 2020-01-03 08 00 40

File: web/src/app/projects/repository/repository.component.html (hunk 2 above)
Comment (webkhushboo) [reply web/src/app/projects/repository/repository.component.html]: I will fix it by alt+shift+F

File: web/src/app/projects/repository/repository.component.scss (hunk 3 above)
Comment (webkhushboo) [reply web/src/app/projects/repository/repository.component.scss:156]: No this one is not causing the issue. there is another css `bottom: -5px` which is causing . I will remove it

**Comment by webkhushboo** [comment]
> When resizing to smallest screen size the layout seems to be a bit weird...
> 
> ## Large
> Screenshot 2020-01-03 08 00 55
> 
> ## Medium
> Screenshot 2020-01-03 08 00 28
> 
> ## Small
> Screenshot 2020-01-03 08 00 40

Its fixed now . Kindly check the screenshot
image

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**